src/svgdevicecontext.cpp

```diff
@@ -427,19 +427,26 @@ void SvgDeviceContext::StartPage()
                                                         .c_str();
     }
 
+    // page rectangle - for debugging
```

Comment:
Should this be removed?

Reply:
I left it for documentation. It is often useful for checking it